x/gamm/twap/hook_test.go

```diff
@@ -0,0 +1,81 @@
+package twap_test
+
+import (
+	"time"
+
+	"github.com/osmosis-labs/osmosis/v10/x/gamm/twap/types"
+)
+
+// TestCreatePoolFlow tests that upon a pool being created,
```

Comment:
Most tests in this file seem to be more functional than unit in nature. While it is useful to have both, I don't think we have any unit tests for hooks.

Do we have an issue for unit testing hooks?

Reply:
I don't think theres any unit tests that makes sense - they should all be functionality tests

Reply:
I guess we could make these more table driven! Definitely very open to figuring out how to make better tests for what we want here!

Reply:
I will make a separate issue for this to discuss further